portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,27 @@
 <!DOCTYPE html>
+<html>
+  <head>
+<!DOCTYPE html>
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>My Portfolio | About Me</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div class="content">
+      <!-- navigation menu -->
+      <div class="header">
+        <a href="aboutme.html">about me</a>
+        <a href="photogallery.html">my photo gallery</a>
+        <a href="recipes.html">recipes i like</a>
+        <a href="https://github.com/jennygaoo">my github</a>
+        <a href="index.html">home page</a>
+      </div>
```

Comment:
this is the same navigation menu across the html files. ideally, we would want one source of truth that all the html pages could reference so that if we do want to make changes to the navigation menu in the future, we don't need to make the edits in multiple places, which also makes it less error prone. 

not something you need to do right now, but i would add a TODO here to consolidate the navigation menu into one place that could be referenced in multiple html files. if you have extra time down the line, you can investigate how to do this.